src/Octoshift/RetryPolicy.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Polly;
+using Polly.Contrib.WaitAndRetry;
+
+namespace OctoshiftCLI
+{
+    public class RetryPolicy
+    {
+        private readonly OctoLogger _log;
+
+        public RetryPolicy(OctoLogger log)
+        {
+            _log = log;
+        }
+
+        public async Task<T> Retry<T>(Func<Task<T>> func, Func<HttpRequestException, bool> filter)
+        {
+            var delay = Backoff.LinearBackoff(TimeSpan.FromMilliseconds(200), retryCount: 5, fastFirst: true);
```

Comment:
While there is nothing wrong with using `Polly.Contrib.WaitAndRetry`, we can also just use the `sleepDurationProvider` of `WaitAndRetryAsync` and not include that package at all. Something like this:

```c#
var retryPolicy = Policy
    .Handle<HttpRequestException>(filter)
    .WaitAndRetryAsync(5, retryCount => retryCount * TimeSpan.FromSeconds(200));
```

 with `sleepDurationProvder` we can cover liner and exponential backoffs. But if you think we're going to end up with more complicated scenarios then it's ok 😃

Reply:
either way is fine with me, 1 less nuget package sounds like a win though so I'll make the change.

Reply:
Yeah exactly, then if need be we can always add it later 😃